src/metpy/calc/__init__.py

```diff
@@ -5,6 +5,7 @@
 
 from .basic import *  # noqa: F403
 from .cross_sections import *  # noqa: F403
+from .exceptions import InvalidSoundingError
```

Comment:
Ugh, bots. Can you change to just:
```suggestion
from .exceptions import *  # noqa: F403
```

Reply:
Ha, OK, I am crossing my fingers that the bots will be happy now...